Cargo.toml

```diff
@@ -17,13 +17,23 @@ log = "0.4"
 clap = "2.33.3"
 regex = "1"
 procfs = "0.8.1"
-actix-web = "3"
-riemann_client = "0.9.0"
+riemann_client = { version = "0.9.0", optional = true }
 hostname = "0.3.1"
 protobuf = "2.20.0"
-serde = { version = "1.0", features = ["derive"] }
-serde_json = "1.0"
-warp10 = "1.0.0"
+serde = { version = "1.0", features = ["derive"], optional = true }
+serde_json = { version = "1.0", optional = true }
+warp10 = { version = "1.0.0", optional = true }
 time = "0.2.25"
 colored = "2.0.0"
 chrono = "0.4.19"
+rs-docker = { version = "0.0.58", optional = true }
```

Comment:
You may use `cargo tree` to find the dependencies.
Maybe it brings older deps ?
![image](https://user-images.githubusercontent.com/2332209/120400994-4bfa8f80-c33f-11eb-9cbb-770f9049f9f6.png)

Reply:
Good one, thanks ! Yes it does. Actually I think I'll use my fork instead: https://github.com/bpetit/rs-docker/
as rs-docker seems to be unmaintained. I'll then update the dependencies so we are even. WDYT ?